.changeset/nervous-moose-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/search-select-field': minor
```

Comment:
Marked as minor change (learned from the releases last few days). Is this right?